src/components/transport_manager/src/transport_adapter/threaded_socket_connection.cc

```diff
@@ -183,7 +183,7 @@ void ThreadedSocketConnection::Send() {
       return;
     }
 
-    if (bytes_sent >= 0) {
+    if (bytes_sent == 0) {
```

Comment:
@LuxoftAKutsan, are you sure that you don't corrupt logic here? I think it's mistake.